### PR TITLE
Check if dir

### PR DESCRIPTION
**Pull request by RobinBurri**
Add Response::check_if_is_dir() 
if the get request is a dir it sends a 404 not found default HTML page.
After we will make a &val object available in response it will send an HTML page
specified in the config.json or 404 not found if not specified.

### CODE REVIEW

**Review by 0xCAF3D0OD — DISMISSED** [review]
OK

File: src/http/Response.cpp [hunk 1]
```diff
@@ -241,4 +255,15 @@ Response::php_handler(const Request &request) const
...
 
+bool
+Response::check_if_is_dir(const std::string &path)
+{
+	struct stat info;
+	if (stat(path.c_str(), &info) != 0)
+	{
+		return false;
+	}
+	return S_ISDIR(info.st_mode);
```
Comment (t-h2o) [inline src/http/Response.cpp:266]: Where is the resource for `S_ISDIR` ?
I didn't find into `man 2 stat`

Comment (t-h2o) [reply src/http/Response.cpp:266]: From `man 7 inode`

**Comment by t-h2o** [comment]
The web server has crashed after 
http://localhost:6060/uploads

**Pull request merged**